tests/testthat/_snaps/class.md

```diff
@@ -4,71 +4,74 @@
       bake_check_class_core(x1, "character", "x1")
     Condition
       Error in `bake_check_class_core()`:
-      ! x1 should have the class(es) character but has the class(es) numeric.
+      ! `x1` should have the class "character" but only has the class "numeric".
 
 ---
 
     Code
       bake_check_class_core(x2, c("POSIXct", "Julian"), "x2")
     Condition
       Error in `bake_check_class_core()`:
-      ! x2 should have the class(es) POSIXct, Julian but has the class(es) POSIXct, POSIXt.
+      ! `x2` should have the class "POSIXct" and "Julian" but only has the classes "POSIXct" and "POSIXt".
 
 ---
 
     Code
       bake_check_class_core(x2, "POSIXct", "x2")
     Condition
       Error in `bake_check_class_core()`:
-      ! x2 has the class(es) POSIXct, POSIXt, but only the following is/are asked POSIXct, allow_additional is FALSE.
+      x `x2` has class "POSIXct" and "POSIXt" but only the following are asked "POSIXct".
```

Comment:
```suggestion
      x `x2` has class "POSIXct" and "POSIXt" but only the following are asked: "POSIXct".
```

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350